Subject: Cron job check for shield-purge before running
Origin: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=461574#10
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=461574
  http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=563065
Forwarded: not-needed
Author: Filippo Giunchedi <filippo@debian.org>,
  Jonathan Niehof <jtniehof@gmail.com>
Last-Update: 2010-04-16

Check for shield-purge before trying to run it, and exit 0 regardless.
Original version of patch (FG) returned 1 if not found; JN updated.
--- a/pam_shield.cron
+++ b/pam_shield.cron
@@ -1,4 +1,5 @@
 #! /bin/sh
 
-/usr/sbin/shield-purge -c /etc/security/shield.conf
+if [ -x /usr/sbin/shield-purge ]; then /usr/sbin/shield-purge -c /etc/security/shield.conf
+fi
 
